resources/asdf-format.org/manifests/transform-1.5.0.yaml

```diff
@@ -660,6 +660,11 @@ tags:
   title: A Scale model.
```

Comment:
```suggestion
  title: A Schechter1D model.
```

Reply:
This is actual the title for the scale model in the previous manifest entry.

Reply:
Ah, OK! :)